qa-core/src/internal-api/fixtures/datasources.ts

```diff
@@ -70,3 +73,50 @@ export const restAPI = (): DatasourceRequest => {
     fetchSchema: false,
   }
 }
+
+export const generateRelationshipForMySQL = (
+  updatedDataSourceJson: any
+): Datasource => {
+  const entities = updatedDataSourceJson!.datasource!.entities!
+  const datasourceId = updatedDataSourceJson!.datasource!._id!
+  const relationShipBody = {
+    ...updatedDataSourceJson.datasource,
+    entities: {
```

Comment:
If there is an issue with typing here - it would be better to cast this structure through `as any as Datasource`  rather than updating the core types - although it would be good to work out what is not valid in this structure for `entities` to not read as a `Table` type.